html.text  setEventListener doesn't work : IE8 compatibility

When I launch my app, now with gwtbootstrap-0.8, i got this error.

Caused by: com.google.gwt.core.client.JavaScriptException: (TypeError) @com.google.gwt.user.client.impl.DOMImpl::setEventListener(Lcom/google/gwt/dom/client/Element;Lcom/google/gwt/user/client/EventListener;)([JavaScript object(118), Java object: org.gwtbootstrap3.client.ui.html.Text@24528244]): Object doesn't support this property or method
    at com.google.gwt.dev.shell.BrowserChannelServer.invokeJavascript(BrowserChannelServer.java:249)
    at com.google.gwt.dev.shell.ModuleSpaceOOPHM.doInvoke(ModuleSpaceOOPHM.java:136)
    at com.google.gwt.dev.shell.ModuleSpace.invokeNative(ModuleSpace.java:576)
    at com.google.gwt.dev.shell.ModuleSpace.invokeNativeVoid(ModuleSpace.java:304)
    at com.google.gwt.dev.shell.JavaScriptHost.invokeNativeVoid(JavaScriptHost.java:107)
    at com.google.gwt.user.client.impl.DOMImpl.setEventListener(DOMImpl.java)
    at com.google.gwt.user.client.DOM.setEventListener(DOM.java:1251)
    at com.google.gwt.user.client.ui.Widget.onAttach(Widget.java:348)
    at com.google.gwt.user.client.ui.AttachDetachException$1.execute(AttachDetachException.java:34)
    at com.google.gwt.user.client.ui.AttachDetachException.tryCommand(AttachDetachException.java:74)
    at com.google.gwt.user.client.ui.Panel.doAttachChildren(Panel.java:129)
    at com.google.gwt.user.client.ui.Widget.onAttach(Widget.java:354)
    at com.google.gwt.user.client.ui.AttachDetachException$1.execute(AttachDetachException.java:34)
    at com.google.gwt.user.client.ui.AttachDetachException.tryCommand(AttachDetachException.java:74)
    at com.google.gwt.user.client.ui.Panel.doAttachChildren(Panel.java:129)

--- a/gwtbootstrap3/src/main/java/org/gwtbootstrap3/client/ui/html/Text.java
+++ b/gwtbootstrap3/src/main/java/org/gwtbootstrap3/client/ui/html/Text.java
@@ -22,6 +22,7 @@ package org.gwtbootstrap3.client.ui.html;
 
 import com.google.gwt.dom.client.Document;
 import com.google.gwt.dom.client.Element;
+import com.google.gwt.event.logical.shared.AttachEvent;
 import com.google.gwt.user.client.ui.HasText;
 import com.google.gwt.user.client.ui.Widget;
 
@@ -43,6 +44,7 @@ import com.google.gwt.user.client.ui.Widget;
 public class Text extends Widget implements HasText {
 
     private final com.google.gwt.dom.client.Text text;
+    private boolean isAttached;
 
     /**
      * Creates the default text node with empty text
@@ -75,5 +77,29 @@ public class Text extends Widget implements HasText {
     @Override
     public String getText() {
         return text.getData();
+    }
+    
+    @Override
+    public boolean isAttached() {
+        return isAttached;
+    }
+
+    @Override
+    protected void onAttach() {
+        if (isAttached()) {
+            throw new IllegalStateException("Text is already attached!");
+        }
+        isAttached = true;
+        onLoad();
+        AttachEvent.fire(this, isAttached);
+    }
+
+    @Override
+    protected void onDetach() {
+        if (!isAttached()) {
+            throw new IllegalStateException("Text is not attached!");
+        }
+        isAttached = false;
+        AttachEvent.fire(this, false);
     }
 }

